pkg/document/json/primitive.go

```diff
@@ -220,3 +220,22 @@ func (p *Primitive) Remove(removedAt *time.Ticket) bool {
 func (p *Primitive) ValueType() ValueType {
 	return p.valueType
 }
+
+// Increase increase integer, long or double.
+func (p *Primitive) Increase(v *Primitive) *Primitive {
+	pType := p.valueType
+	vType := v.valueType
+	if pType != Integer && pType != Long && pType != Double && vType != Integer && vType != Long && vType != Double {
```

Comment:
If pType or vType is not a numeric type, we will have to judge it as an error condition. It would be less confusing to extract a function that returns whether it is numeric or not(`IsNumericType`). And It would be nice to add test code for edge cases as well.

Reply:
```go
if (pType != Integer && pType != Long && pType != Double) || (vType != Integer && vType != Long && vType != Double)
```